DiscordCardLinker/CardBot.cs

```diff
@@ -130,19 +160,19 @@ public class CardBot {
 			 * Only add the card key once.
 			 */
 			if(!collection.ContainsKey(key)) {
-				Console.WriteLine("Adding Key: ["+key+"]");
+				//Console.WriteLine("Adding Key: ["+key+"]");
```

Comment:
Please do not comment out debugging information. This information is critical for debugging the container service if it is in a frozen state.

Reply:
Apologies, forgot to restore this (it was causing a delay of a few minutes each run).